doc/build-netbsd.md

```diff
@@ -29,8 +30,7 @@ See [dependencies.md](dependencies.md) for a complete overview.
 
 ### Building BerkeleyDB
 
-BerkeleyDB is only necessary for the wallet functionality. To skip this, pass
-`--disable-wallet` to `./configure` and skip to the next section.
+BerkeleyDB is only necessary for legacy wallet functionality.
```

Comment:
Nit: I think it's Berkeley DB, with a space. 🤔

Reply:
In the netbsd doc we are not mentioning anywhere the thing about "sqlite3 enables descriptor based wallet". Maybe that line could be added somewhere?

Reply:
> Nit: I think it's Berkeley DB, with a space. 🤔

Done in next push.